tests/utils/test_helpers.py

```diff
@@ -248,6 +248,16 @@ def test_cross_downstream(self):
         for start_task in start_tasks:
             six.assertCountEqual(self, start_task.get_direct_relatives(upstream=False), end_tasks)
 
+    def test_chain(self):
+        dag = DAG(dag_id='test_chain', start_date=datetime.now())
+        [t1, t2, t3, t4, t5, t6] = [DummyOperator(task_id='t{i}'.format(i=i), dag=dag) for i in range(1, 7)]
+        helpers.chain(t1, [t2, t3], [t4, t5], t6)
+
+        self.assertCountEqual([t2, t3], t1.get_direct_relatives(upstream=False))
+        self.assertEqual([t4], t2.get_direct_relatives(upstream=False))
+        self.assertEqual([t5], t3.get_direct_relatives(upstream=False))
+        self.assertCountEqual([t4, t5], t6.get_direct_relatives(upstream=True))
+
```

Comment:
We need more tests showing what happens in case two neighbouring iterables are not the same length.

Reply:
We discuss above, if we decide add raise Error if list not in the same size, I will add more unit test, thanks

Reply:
I think tests are needed regardless - they should test the behaviour that is description. - but showing possibly different behaviour (swallowing if we decide to do so or error).

Reply:
I add more test on different length of iterable and between not two iterable